### DISCUSSION TITLE
WINDOW_MAXIMIZE_REQUEST and WINDOW_MINIMIZE_REQUEST

### DISCUSSION THREAD

**Discussion by ghost** [tauri-apps/tauri#6020] [Q&A]
I know there's TauriEvent.WINDOW_RESIZED 

but it does not help

is there extra WINDOW_MINIMIZE/MAXIMIZE event?

kind of WINDOW_CLOSE_REQUEST but for maximize

**Comment by FabianLars** [comment]
No there isn't, at least not yet. So far people used the aforementioned resize and moved events to workaround this.

**Reply by ghost** [reply]
So bad. Would be really good if you guys could add `WINDOW_MAXIMIZE_REQUEST` and `WINDOW_MINIMIZE_REQUEST` for syncing it with the application's internal state
